frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte

```diff
@@ -139,18 +139,22 @@
 			on:consider={handleMoved}
 			on:finalize={handleDropped}
 		>
-			{#each elements as element (element.id)}
-				<ElementContainer
-					bind:element
-					bind:editId
-					bind:dragEnabled
-					bind:showConfirmDelete
-					{addElement}
-					{chartOptions}
-					{sliceOptions}
-					report={data.report}
-				/>
-			{/each}
+			{#await chartOptions then chartOptions}
```

Comment:
Do we always want to have to await these even if we are just viewing? Just viewing we don't need to make these fetch calls.